src/interactor.tsx

```diff
@@ -7,9 +7,8 @@ import { TxButton, TxGroupButton } from './substrate-lib/components';
 const argIsOptional = (arg) =>
   arg.type.toString().startsWith('Option<');
 
-function Main (props) {
+function Main ({ accountPair } : {accountPair: any}) {
```

Comment:
It's a bit weird to have this half-typed argument lying around when most of the codebase is still not typified. I don't think we should change it back to `any`, but just something to consider next time where we should either typify everything all the way, or don't touch anything until we're ready to tackle it.

Reply:
Done